# DP: gm2.texi: Define our own gcctabopt macro, needed for dfsg builds.

--- a/src/gcc/doc/gm2.texi
+++ b/src/gcc/doc/gm2.texi
@@ -51,6 +51,25 @@ man page gfdl(7).
 @versionsubtitle
 @author Gaius Mulley
 
+@macro gcctabopt{body}
+@code{\body\}
+@end macro
+@macro gccoptlist{body}
+@smallexample
+\body\
+@end smallexample
+@end macro
+@c Makeinfo handles the above macro OK, TeX needs manual line breaks;
+@c they get lost at some point in handling the macro.  But if @macro is
+@c used here rather than @alias, it produces double line breaks.
+@iftex
+@alias gol = *
+@end iftex
+@ifnottex
+@macro gol
+@end macro
+@end ifnottex
+
 @page
 @vskip 0pt plus 1filll
 Published by the Free Software Foundation @*
